tests/integration/components/feed-item-test.js

```diff
@@ -9,7 +10,43 @@ test('it renders', function(assert) {
   // Set any properties with this.set('myProperty', 'value');
   // Handle any actions with this.on('myAction', function(val) { ... });
 
-  this.render(hbs`{{feed-item}}`);
+  let article = {
+    author: {
+      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+      username: 'Alon Bukai'
+    },
+    createdAt: moment('1995-12-25'),
+    favoritesCount: 9000,
+    body: 'This is feed item content',
+    slug: 'feed-item-title',
+    title: 'Feed Item Title'
+  };
 
-  assert.equal(this.$().text().trim(), '');
+  this.set('article', article);
+  this.render(
```

Comment:
Yea I understand what you mean now. 
If you haven't taken a look at Prettier I recommend you check it out.
There is no reason for us to argue over syntax and style when it can all be decided for us.